### PR TITLE
Wrap piping from form-data to request in a try-catch block

### PR DESCRIPTION
**Pull request by harryi3t**
## PR Checklist:
- [x] I have run `npm test` locally and all tests are passing.
- [x] I have added/updated tests for any new behavior.
       
- [x] If this is a significant change, an issue has already been created where the problem / solution was discussed: [request](https://github.com/request/request/issues/1561#issuecomment-353976548), [form-data](https://github.com/form-data/form-data/issues/375), [combined-stream](https://github.com/felixge/node-combined-stream/issues/37)
       

## PR Description
When the form-data size is too big, an exception is thrown because of the way an internal module has been written (combined-stream)
This PR adds a temporary patch (a try-catch block) to fix this issue.

@kunagpal ~Builds are failing on master as well.~ It was happening coz of an ipV6 test. Have disabled the test for Travis

### CODE REVIEW

**Comment by kunagpal** [comment]
@harryi3t Could you update `tough-cookie` to the latest version?

File: tests/test-form-data-postman.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  s.listen(0, function () {
+    t.end()
+  })
+})
+
+tape('large formData should return an error', function (t) {
+  request({
+    uri: s.url + path,
+    method: 'post',
+    formData: {foo: new Array(3e3).fill('bar')}
```
Comment (kunagpal) [inline tests/test-form-data-postman.js]: 👍

**Comment by harryi3t** [comment]
CC @shamasis

**Comment by shamasis** [comment]
LGTM - update tough cookie here or @kunagpal do it on separate branch and merge to develop and here. Whichever is fastest

**Comment by harryi3t** [comment]
@shamasis we have already the latest version of tough-cookie (2.3.3) which does not have any security vulnerability 
Somehow NSP is detecting 2.3.2 (which has a security vulnerability) and failing :(

**Comment by czardoz** [comment]
@harryi3t The dependency is not pinned in `package.json`, remove the `~` and it should work.

**Comment by czardoz** [comment]
Also, you'd want to update the lock file

**Comment by harryi3t** [comment]
@czardoz 
We have `~2.3.3` in our package.json file. 
Doesn't this mean install the latest patch >= `2.3.3` and < `2.4.0`?

**Comment by harryi3t** [comment]
@czardoz Updating the `package-lock.json` file did the trick 🎉 
Thanks!

**Pull request merged**